README.md

```diff
@@ -33,7 +33,7 @@ Change summary
 | R97        | 20 -> 13 (5 -> 8 BTK) | 12 -> 8 (9 -> 13 BTK)  | 10 -> 7 (10 -> 15 BTK) |
 | R101       | 25 -> 17 (4 -> 6 BTK) | 17 (6 BTK) (_vanilla_) | 12 (9 BTK) (_vanilla_) |
 | R201       | 25 -> 17 (4 -> 6 BTK) | 17 (6 BTK) (_vanilla_) | 12 (9 BTK) (_vanilla_) |
-| Flatline   | 30 -> 20 (4 -> 5 BTK) | 20 (5 BTK) (_vanilla_) | 15 (7 BTK) (_vanilla_) |
+| Flatline   | 30 -> 20 (4 -> 5 BTK) | 17 (6 BTK)             | 15 (7 BTK) (_vanilla_) |
```

Comment:
Since it's not vanilla anymore, we write the change:

```
20 -> 17 (5 -> 6 BTK)
```

I can't suggest this because the table width needs to be correct